.github/src/sdk-types.js

```diff
@@ -0,0 +1,50 @@
+// @ts-check
+
+/**
+ * @typedef {'azure-sdk-for-go' | 'azure-sdk-for-java' | 'azure-sdk-for-js' | 'azure-sdk-for-net' | 'azure-sdk-for-python'} SdkName
+ */
+
+/**
+ * SDK labels mapping for breaking change labels
+ * @type {Object.<SdkName, {
+ *   breakingChange: string | undefined,
+ *   breakingChangeApproved: string | undefined,
+ *   breakingChangeSuppression: string | undefined,
+ *   breakingChangeSuppressionApproved: string | undefined
```

Comment:
```suggestion
 *   breakingChange: string,
 *   breakingChangeApproved: string,
 *   breakingChangeSuppression: string,
 *   breakingChangeSuppressionApproved: string
```

Do these need to support undefined?  All keys appear to be defined in all objects below.